ssg/build_sce.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import os.path
+import json
+
+
+from .build_yaml import Rule, DocumentationNotComplete
+from .constants import MULTI_PLATFORM_LIST
+from .jinja import process_file_with_macros
+from .rule_yaml import parse_prodtype
+from .rules import get_rule_dir_id, get_rule_dir_sces, find_rule_dirs_in_paths
+from . import utils
+
+def load_sce_and_metadata(file_path, local_env_yaml):
+    raw_content = process_file_with_macros(file_path, local_env_yaml)
+
+    metadata = dict()
+    sce_content = []
+
+    for line in raw_content.split("\n"):
+        if line.startswith('##ssg##'):
```

Comment:
@jan-cerny: Currently the list includes `platform`, `check-import`, `check-export` and eventually `complex-check` (in an internally merged changeset that includes SCE templating support). They are sometimes of more complex types (similar to `variables` in the test suite) including lists with interior `=` assignments.

Since I preferred the metadata first ahead of the shebang and also grouped together (logically, SCE will likely have more metadata headers than most other interpreted content types -- and when reading, you care more about the SCE metadata or the check content than the shebang), I chose a unique prefix and stripped it out of the resulting output file.

I was also kinda leaning towards doing so consistently across all formats (using `##ssg##` and stripping it).

But I was curious what the thoughts of the group were :-)

If you don't like it, I can drop it.

Reply:
Good point but indeed it isn't consistent with the other content types.

Reply:
ACK will update :)

Reply:
Note that this has been removed.